src/routes/postNoSegments.js

```diff
@@ -0,0 +1,78 @@
+const db = require('../databases/databases.js').db;
+const getHash = require('../utils/getHash.js');
+const isUserVIP = require('../utils/isUserVIP.js');
+const logger = require('../utils/logger.js');
+
+module.exports = (req, res) => {
+    // Collect user input data
+    let videoID = req.body.videoID;
+    let userID = req.body.userID;
+    let categorys = req.body.categorys;
+
+    // Check input data is valid
+    if (!videoID 
+        || !userID 
+        || !categorys 
+        || !Array.isArray(categorys) 
+        || categorys.length === 0
+    ) {
+        res.status(400).json({
+            status: 400,
```

Comment:
I think it's redundant to include the status here. Just the message should be needed.

Reply:
as text or body.message?

Reply:
i went with body.message re https://github.com/ajayyy/SponsorBlockServer/issues/126